src/content/docs/en/guides/images.mdx

```diff
@@ -145,6 +142,22 @@ When using local images in their original aspect ratio, the `width` and `height`
 
 However, both of these properties are required for remote images and images stored in your `public/` folder as Astro is unable to analyze these files.
 
+##### densities
+
+<Since v="3.3.0" /> <Badge>Experimental</Badge>
+
+A list of pixel densities to generate for the image. By default, no other densities are generated. Densities that are equals to widths larger than the original image will be ignored, as to avoid upscaling the image.
+
+This value will be used to generate a `srcset` attribute on the `<img>` tag.
```

Comment:
``````suggestion
A list of pixel densities to generate for the image. 

If provided, this value will be used to generate a `srcset` attribute on the `<img>` tag. Do not provide a value for `widths` when using this value.

Densities that are equals to widths larger than the original image will be ignored to avoid upscaling the image.

```astro
---
import { Image } from 'astro:assets';
import myImage from "../assets/my_image.png";
---
<Image src={myImage} densities={[1x, 2x, 3x]} alt="A description of my image." />
```
``````
Can we add an astro/html output example here? Something like above, with the output?